src/patternfly/components/DataList/data-list.scss

```diff
@@ -24,11 +24,17 @@ $pf-data-list--breakpoint--mobile: 380px;
   --pf-c-data-list__item--before--Width: var(--pf-global--BorderWidth--lg);
   --pf-c-data-list__item--before--Transition: var(--pf-global--Transition);
   --pf-c-data-list__item--before--ZIndex: var(--pf-global--ZIndex--xl);
+  --pf-c-data-list__item--before--Top: 0;
+  --pf-c-data-list__item--before--Height: 100%;
+  --pf-c-data-list__item-item--before--Top: calc(var(--pf-c-data-list__item-item--BorderTopWidth) * -1);
+  --pf-c-data-list__item-item--before--Height: calc(100% + var(--pf-c-data-list__item-item--BorderTopWidth));
 
   // Cell
   --pf-c-data-list__cell--PaddingTop: var(--pf-global--spacer--xl);
   --pf-c-data-list__cell--PaddingBottom: var(--pf-global--spacer--xl);
-  --pf-c-data-list__cell--MarginLeft: var(--pf-global--spacer--xl);
+  --pf-c-data-list__cell-cell--PaddingTop: 0;
+  --pf-c-data-list__cell-cell--md--PaddingTop: var(--pf-c-data-list__cell--PaddingTop);
```

Comment:
Are we still going ahead with assigning variables to a previously stated variable? Or could that cause a breaking change?